pages/instructorQuestionManualGrading/instructorQuestionManualGradingUnlockInstanceQuestions.js

```diff
@@ -0,0 +1,23 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const {sqlDb, sqlLoader} = require('@prairielearn/prairielib');
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+router.get('/', (req, res, next) => {
```

Comment:
This seems to be an explicitly state-modifying query and so should be a POST?

Reply:
This endpoint was eliminated, as the locking feature logic was not necessary.